test/cs_guide/resources/resources_test.exs

```diff
@@ -2,200 +2,199 @@ defmodule CsGuide.ResourcesTest do
   use CsGuide.DataCase
 
   alias CsGuide.Resources
+  alias CsGuide.Resources.Brand
+  alias CsGuide.Fixtures
+
+  @create_brand Fixtures.create_brand()
 
   describe "venues" do
     alias CsGuide.Resources.Venue
 
-    @valid_attrs %{phone_number: "some phone_number", postcode: "some postcode", venue_name: "some venue_name"}
-    @update_attrs %{phone_number: "some updated phone_number", postcode: "some updated postcode", venue_name: "some updated venue_name"}
+    @valid_attrs %{
+      phone_number: "some phone_number",
```

Comment:
**TODO** (_on post launch iteration_): **`phone_number`** should be refactored to use `Fields.phone_number` https://github.com/dwyl/fields/issues/17 which will do pre-storage validation. (_which we appear to not have right now, so people could submit "blah!" or `<script>` as the venue phone number and the App would accept it blindly ..._)